cmd/podman/cleanup.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/urfave/cli"
+)
+
+var (
+	cleanupFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "Cleans up all containers",
+		},
+		LatestFlag,
+	}
+	cleanupDescription = `
+   podman container cleanup
+
+   Cleans up mount points and network stacks on one or more containers from the host.  The container name or ID can be used. This command is used internally when running containers, but can  also be used if container cleanup has failed when a container exits.
```

Comment:
Looks like you have two spaces in `can  also`.

Reply:
Fixed